src/app/orbit-counts/orbit-counts.component.ts

```diff
@@ -16,16 +16,25 @@ export class OrbitCountsComponent implements OnInit {
   }
 
   countByType(type: string): number {
-	let count = 0;
-	if (this.satellites) {
-	  for (let i = 0; i < this.satellites.length; i++) {
-		 if (this.satellites[i].type === type) {
-			count++;
-		 }
-	  }
-	}
-	return count;
- }
+	
 
+for (let i = 0; i < this.satelliteList.length; i++){
+      let type = this.satelliteList[i].type; 
```

Comment:
There is no variable name `satelliteList` in this component, so this is causing a lot of errors. Where is this variable coming from?

Reply:
I will look at it. Also, I got a notification that Assignment #3 has been graded. Did I get 0 points? What is wrong with it? Could you please give me some feedback? Thank you